hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/ExternalConditionEval.java

```diff
@@ -75,6 +99,14 @@ public void setValue(String value) {
         this.value = value;
     }
 
+    public Event getEvent() {
+        return event;
+    }
+
+    public void setEvent(Event event) {
+        this.event = event;
+    }
```

Comment:
If you keep the 'event'  and 'valueType' fields, this setter should probably set the valueType to EVENT for safe deserialization.